src/JsonSchema/Validator.php

```diff
@@ -26,6 +26,7 @@ class Validator extends BaseConstraint
     const SCHEMA_MEDIA_TYPE = 'application/schema+json';
 
     const ERROR_NONE                    = 0x00000000;
+    const ERROR_ALL                     = ~self::ERROR_NONE;
```

Comment:
That way you can do fancy things like fetching errors for `(Validator::ERROR_ALL & ~Validator:ERROR_I_NEVER_WANT_TO_SEE)`. We only have two error contexts right now, but if that changes later it'll come in handy.